modules/swagger-models/src/main/java/io/swagger/models/Swagger.java

```diff
@@ -272,6 +272,11 @@ public void addProduces(String produces) {
         return sorted;
     }
 
+    @JsonIgnore
+    public Map<String, Path> getPathsMap() {
```

Comment:
`getPaths` return `Map` sorted by `key` and this one return original `Map` with insertion order. I can remove it if it is not ok.